test/compilable/test21719.d

```diff
@@ -0,0 +1,21 @@
+// https://issues.dlang.org/show_bug.cgi?id=21719
+
+struct S
+{
+    auto f()
+    {
+    } // inferred to be @safe @nogc pure nothrow
+}
+
+class C
+{
+    auto f()
+    {
+    } // inferred to be @system (only)
```

Comment:
This was the code in the bug report.
Reading the comment as part of this test, without the issue context, is confusing, as it's not describing what's actually happening, but what used to.
It's good to use the PR's code as a base for the test case, but it is not always enough. In this case, this is only testing `@safe`, not the other attributes.

Reply:
Well, there was another test that I had to modify which was basically the same except that all attributes were tested so I thought that I don't have to be more religious than the pope.